kohttp/src/main/kotlin/io/github/rybalkinsd/kohttp/ext/ResponseExt.kt

```diff
@@ -77,25 +78,23 @@ data class EagerResponse(
  */
 data class Header(val name: String, val value: String)
 
-
-
 /**
  * Returns Response Body as String.
  *
  * @return Response body as a `String?`.
+ * !! this function will close `ResponseBody`
```

Comment:
May be it is better to write:
```suggestion
 * <p>
 * Note: This function will close `ResponseBody`
 * </p>
```